.gitpod.yml

```diff
@@ -6,6 +6,9 @@ ports:
     onOpen: ignore
 
 tasks:
+  - name: install pre-commit and initialize it
+    init: pip3 install pre-commit && pre-commit install
+    command: pre-commit run --all-files
```

Comment:
I'm not sure that running `pre-commit` when workspaces start is valuable. In other words, if we want to get the users feedback earlier, so that they know when something will fail our check, what do you think of doing so on the [pre-commit hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks#:~:text=The%20pre%2Dcommit,on%20new%20methods.)?

Reply:
@kylos101 `pre-commit install` installs the git hooks.[^1]

[^1]: https://pre-commit.com/#3-install-the-git-hook-scripts

Reply:
You rock, thank you @axonasif ! I should have read more thoroughly, my apologies. :pray: